ext/include/opentelemetry/ext/http/server/socket_tools.h

```diff
@@ -14,6 +14,7 @@
 #pragma once
 
 #include <algorithm>
+#include <atomic>
```

Comment:
Is this necessary?

Reply:
Yes, there is atomic variable inside, and compilation fails without that. This used to get silently passed earlier due to other files including this header.